core/src/main/java/org/mule/runtime/core/internal/policy/CompositeOperationPolicy.java

```diff
@@ -130,4 +137,11 @@ public CoreEvent process(CoreEvent event) throws MuleException {
       return error(e);
     }
   }
+
+  private void logOperationEvent(EventContext eventContext, Message message) {
+    if (LOGGER.isTraceEnabled()) {
+      LOGGER.trace("Executing operation\n" + eventContext.toString() + "\n"
```

Comment:
Change `Executing operation` for something indicating it it's about to execute or already executed, since that seems to be the use case.